aiohttp/web_protocol.py

```diff
@@ -575,13 +575,13 @@ async def start(self) -> None:
                 else:
                     task = loop.create_task(coro)
                 try:
-                    resp, reset = await task
+                    resp, reset = await task  # type: ignore[possibly-undefined]
```

Comment:
This looks like another regression? How can this be undefined?

Reply:
Hmm, seems to have resulted from the Python version change. Still don't see how it can be undefined though..